tests/fixture/pageObject/playwright/PollyObject.ts

```diff
@@ -0,0 +1,160 @@
+import { Page } from '@playwright/test';
+import { MODE, Polly } from '@pollyjs/core';
+import { API } from '@pollyjs/node-server';
+import FSPersister from '@pollyjs/persister-fs';
+import { includes, isObject } from 'lodash';
+import path from 'path';
+import { PlaywrightAdapter } from 'polly-adapter-playwright';
+
+import { ignoredHeaders, ignoredPayloads } from './senstiveDataConfig';
+import { CustomFSPersister } from './CustomFSPersister';
+
+interface PollyOptions {
+    har: string;
+    mode: MODE;
+    page: Page;
+    storeUrl: string;
+    devMode: boolean;
+}
+
+export class PollyObject {
+    /**
+     * @internal
```

Comment:
Move this docblock above `PollyObject` if your intention is to leave a comment against it, e.g.:

```ts
/**
 * @internal
 * This file is only accessible to PlaywrightHelper.
 */
export class PollyObject {
```
Otherwise, you're actually commenting on `private polly: Polly | undefined;`

Reply:
Thank you for reminding me. All occurrences have been corrected.